HomeWork/src/main/java/HomeWork6/Employee.java

```diff
@@ -0,0 +1,57 @@
+package HomeWork6;
+
+public class Employee {
+    private String username;
+    private String position;
+    private String email;
+    private String numberPhone;
```

Comment:
Would be better to rename `numberPhone` to `phoneNumber`. Please rename getter and setter for this field as well